src/components/Search/Form.js

```diff
@@ -0,0 +1,28 @@
+import { render } from '../../shared/dom';
```

Comment:
Gonna leave the UI "comment" here:

1. Input doesn't look like on designs ~ lack of styling, wrong colors, lack of icon
2. It's too big
3. Search something should disapear when user will start typing